ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/index.scss

```
@@ -0,0 +1,17 @@
+.advanced-gas-fee-input-subtext {
+  display: flex;
+  align-items: center;
+  margin-top: 2px;
+  color: $ui-4;
+  font-size: 12px;
```

Similarly to colors we should avoid using staic pixel values for font-size. Use design system variable:`$font-size-h7` or mixin: `@include H7`.
```suggestion
  font-size: $font-size-h7;
```